dist/OutSystemsUI.js

```diff
@@ -18346,10 +18350,29 @@ var Providers;
                             super(uniqueId, new SingleDate.FlatpickrSingleDateConfig(configs));
                             this._isUpdatedInitialDateByClientAction = false;
                         }
+                        _checkInitialDate() {
+                            let clearPlatformInput = false;
+                            if (OSFramework.OSUI.Helper.Dates.IsNull(this.configs.InitialDate)) {
+                                if (this.datePickerPlatformInputElem.value !== OSFramework.OSUI.Constants.EmptyString &&
+                                    OSFramework.OSUI.Helper.Dates.IsValid(this.datePickerPlatformInputElem.value)) {
+                                    this.configs.InitialDate = new Date(this.datePickerPlatformInputElem.value).toString();
+                                }
+                                else {
+                                    clearPlatformInput = true;
+                                }
+                            }
+                            else if (this.datePickerPlatformInputElem.value !== OSFramework.OSUI.Constants.EmptyString) {
+                                clearPlatformInput = true;
+                            }
+                            if (clearPlatformInput) {
+                                OSFramework.OSUI.Helper.Dom.Attribute.Set(this.datePickerPlatformInputElem, OSFramework.OSUI.GlobalEnum.HTMLAttributes.Value, OSFramework.OSUI.Constants.EmptyString);
+                            }
+                            this.prepareConfigs();
+                        }
                         onDateSelectedEvent(selectedDates) {
                             let _selectedDate = '';
                             if (selectedDates.length > 0) {
-                                _selectedDate = this.provider.formatDate(selectedDates[0], this.flatpickrOpts.dateFormat);
+                                _selectedDate = this.provider.formatDate(selectedDates[0], this.provider.enableTime ? 'Y-m-d H:i:S' : 'Y-m-d');
```

Comment:
After spoke with @joselrio , we decided to keep this format as string and not on a Enum because this a specific use case on DatePicker SingleDate.